### PR TITLE
Sync again with nix.dev

### PR DESCRIPTION
**Pull request by am-on**
- Add missing tutorials from nix.dev
- Move `Install Nix` tutorial to be included in Download page
- Fix link to PatchELF

### CODE REVIEW

File: blog/index.tt [hunk 1]
```diff
@@ -1526,7 +1526,7 @@ for the next edition or look at the
 <p>lorri is based around fast direnv integration for robust CLI and editor integration.</p>
 <p><a href="http://github.com/target/lorri">Try it out</a></p>
 <ul>
-<li><a href="https://nixos.org/releases/patchelf/patchelf-0.10/">PatchELF 0.10 released</a></li>
+<li><a href="https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/">PatchELF 0.10 released</a></li>
```
Comment (jtojnar) [inline blog/index.tt:1529]: I get `Error: [object Object]` here

Comment (am-on) [reply blog/index.tt:1529]: Strange, the link works for me in firefox on macos. [https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/](https://releases.nixos.org/?prefix=patchelf/patchelf-0.10/). 

Screenshot 2021-12-11 at 09 59 00

Comment (jtojnar) [reply blog/index.tt:1529]: I was caused by uBlock, sorry for the noise.

**Comment by garbas** [comment]
@am-on Sorry for hijacking this PR, but you added one tutorial to the list from nix.dev and I just add the rest.

**Comment by garbas** [comment]
I'm going to close this PR and push the branch directly to the repo so that preview link will be created.

**Comment by garbas** [comment]
https://github.com/NixOS/nixos-homepage/pull/780

**Pull request closed without merging**